source/db/en-projects.json

```diff
@@ -6254,5 +6254,37 @@
       }
     ],
     "slug": "android-password-store"
+  },
+  {
+    "development_stage": "released",
+    "description": "Self-hosted, decentralized OStatus-compatible social network.",
+    "license_url": "https://github.com/tootsuite/mastodon/blob/master/LICENSE",
+    "logo": "mastodon.svg",
+    "notes": "Policies may vary between instances, choose one that fits you.",
+    "privacy_url": "",
+    "source_url": "https://github.com/tootsuite/mastodon",
+    "name": "Mastodon",
+    "tos_url": "",
+    "url": "https://joinmastodon.org",
+    "wikipedia_url": "https://en.wikipedia.org/wiki/Mastodon_(software)",
+    "protocols": [
+      "OStatus",
```

Comment:
We should probably add `ActivityPub` into this list; I expect more and more of the applications in the social category will be moving to support it since it's a W3C recommendation